rskj-core/build.gradle

```diff
@@ -1,10 +1,127 @@
 plugins {
     id 'application'
+    id 'checkstyle'
+    id "com.diffplug.spotless" version "6.13.0"
 }
 
 apply plugin: 'maven-publish'
 apply plugin: 'jacoco'
 
+import java.nio.file.Paths
+
+// Define static timestamp
+def staticTimestamp = '2024-09-03T02:17:00'
```

Comment:
There is also the option to use [ratchetFrom](https://github.com/rsksmart/powpeg-node/pull/250/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R176), which only formats from a given git tag

Reply:
Thank you for the suggestion! I considered using `ratchetFrom` to format only changes from a specific git tag. However, [the last tag](https://github.com/rsksmart/rskj/tags) in the repository is from July 10, which is quite a while ago. Using ratchetFrom would include all changes made since that tag, which would affect files modified after that date—something we want to avoid in this case